### DISCUSSION TITLE
Quickie Websites

### DISCUSSION THREAD

**Discussion by rahaaatul** [rahaaatul/automate-flows#9] [Ideas]
Request missing websites here.

**Comment by hdbordercollie** [comment]
Searxng, the search engine, and Dumb, the Genius frontend, for lyrics. By the way, how would one go about setting a custom default search engine?

**Reply by rahaaatul** [reply]
Can you provide the links to the sites?

> How would one go about setting a custom default search engine?

You'd have to edit the `web dialog` block, replace `@google ` (mind the space at the end, it's crucial) with some other sites from the `SEARCH_ENGINE_LIST` variable block.

**Comment by hdbordercollie** [comment]
Thanks! Maybe the instructions should be added to the README or Flow description.

Dumb and Searxng have several instances as they're open-source projects, but I use some of the most popular, dumb.ducks.party and PaulGo

You can also just add Genius instead of Dumb if you want.

https://dumb.ducks.party/search?q=
https://paulgo.io/search?q=

**Reply by rahaaatul** [reply]
Hi, I've already added `genius` and `searx` but I'm reluctant on adding multiple instances of same site. Does `https://searx.org` work for you?

I'll add `dumb` on next version.

**Reply by hdbordercollie** [reply]
Awesome possum. Works just as well.

**Reply by rahaaatul** [reply]
Updated to [Version 5](https://github.com/rahaaatul/automate-flows/tree/main/Quickie#-release-notes-5). 

Also, You can find `searx_paulgo` for `https://paulgo.io` in the new version.
